History.md

```diff
@@ -1,3 +1,14 @@
+# 0.11.3 / 2017-11-04
```

Comment:
This file is updated automatically on release using [Luzifer/git-changerelease](https://github.com/Luzifer/git-changerelease), please do not change it within a PR.